tests/test_report.py

```diff
@@ -0,0 +1,85 @@
+from unittest.mock import patch
+
+from freezegun import freeze_time
+
+from dsc.report import FinalizeReport
+from dsc.workflows.base import WorkflowEvents
+
+
+@freeze_time("2025-01-01 09:00:00")
+def test_finalize_report_init_success(workflow_events_finalize):
```

Comment:
Would we ever instantiate a report this way or is it always expected to be `.from_workflow`?

Reply:
Good question! I think, yes, it would always be `.from_workflow`, but I think the `__init__` method can be handy for testing?